_pages/en_US/bannerbomb3.txt

```diff
@@ -22,43 +22,41 @@ If you have a Taiwanese console (indicated by a T after the system version), fol
 ### What You Need
 
 * Your `movable.sed` file from completing [Seedminer](seedminer)
+* The latest release of [BannerBomb3](https://github.com/zoogie/Bannerbomb3/releases/latest) (the Release `.zip` file)
```

Comment:
it doesn't look like we're doing anything with this file in the instructions

Reply:
Resolved.